packages/CLAPACK/Makefile

```diff
@@ -19,17 +19,25 @@ $(SRC)/lapack_WA.bc:  $(SRC)/Makefile
 	# On target it gets rewritten by pywasmcross to the full patch of
 	# blas_WA.bc, lapack_WA.bc which are linked statically in scipy
 	# in each module that needs them.
-	emmake make -C CLAPACK-WA/ -j $${PYODIDE_JOBS:-3}
+	emmake make -C CLAPACK-WA/ -j $${PYODIDE_JOBS:-3} blaslib lapacklib
 
-
-$(SRC)/Makefile:
+clapack.tgz:
 	wget -q -O clapack.tgz http://www.netlib.org/clapack/clapack.tgz
 	md5sum --quiet --check checksums || (rm -f clapack.tgz; false)
+
+$(SRC)/Makefile: clapack.tgz
 	tar xzf clapack.tgz
 	rm -rf $(SRC)
 	mv CLAPACK-$(CLAPACK_VERSION) $(SRC)
+	chmod -R o+rx $(SRC)
```

Comment:
Why do we need this?

Reply:
Well we don't *need* it, but it makes life much easier if you build in docker, since it would not be you who owns the content and it defaults to `550`.

Reply:
Right, fair enough. Maybe add a comment about it? Otherwise feel free to merge.